cellprofiler/modules/threshold.py

```diff
@@ -1,15 +1,13 @@
 # coding=utf-8
 
 """
-Apply Threshold
+Threshold
```

Comment:
Can you remove some of the equal signs below to be equal in length to the word Threshold?